### PR TITLE
vol_clone_wipe: Case update

### PR DESCRIPTION
**Pull request by will-Do**
1. Fail case if the testing pool is exist, and it's no need to check volume
then. And check if the new create volume exist in the testing pool
before the next step testing.
2. Adjust the pre-check order to skip the case ASAP.
3. And rename an instance name and the pool/vol name to make them more readable.

Signed-off-by: Yanbing Du <ydu@redhat.com>

### CODE REVIEW

File: libvirt/tests/src/virsh_cmd/volume/virsh_vol_clone_wipe.py [hunk 1]
```diff
@@ -78,35 +58,52 @@ def run(test, params, env):
...
+                                vol_format=vol_format,
+                                capacity=vol_capability,
+                                allocation=None,
+                                pool_name=pool_name)
         elif vol_format == 'partition':
-            vol_name = libv_vol.list_volumes().keys()[0]
-            logging.debug("Partition %s in disk pool is volume" % vol_name)
+            if not libvirt_vol.list_volumes().keys():
+                raise error.TestError("No volume in pool: %s" % pool_name)
+                vol_name = libvirt_vol.list_volumes().keys()[0]
```
Comment (Hao-Liu) [inline libvirt/tests/src/virsh_cmd/volume/virsh_vol_clone_wipe.py]: I think these two lines are over-indented.

**Comment by Hao-Liu** [comment]
I think we'd better add a brief description of what we did and why in the commit message

File: libvirt/tests/src/virsh_cmd/volume/virsh_vol_clone_wipe.py (hunk 1 above)
Comment (will-Do) [inline libvirt/tests/src/virsh_cmd/volume/virsh_vol_clone_wipe.py]: Yes, will update it, thx

**Comment by will-Do** [comment]
Thx @Hao-Liu

File: libvirt/tests/src/virsh_cmd/volume/virsh_vol_clone_wipe.py [hunk 2]
```diff
@@ -78,35 +61,50 @@ def run(test, params, env):
...
+    # Choose an algorithm randomly
+    if wipe_algorithms:
+        alg = random.choice(wipe_algorithms.split())
+    else:
+        alg = random.choice(valid_algorithms)
+
+    libvirt_pvt = utlv.PoolVolumeTest(test, params)
+    libvirt_pool = libvirt_storage.StoragePool()
+    if libvirt_pool.pool_exists(pool_name):
+        raise error.TestError("Pool '%s' already exist", pool_name)
```
Comment (kylazhang) [inline libvirt/tests/src/virsh_cmd/volume/virsh_vol_clone_wipe.py]: The above judgement should be in try as if pool already exists, we need to clean the env for following case

**Comment by kylazhang** [comment]
Looks good to me for the left part.

File: libvirt/tests/src/virsh_cmd/volume/virsh_vol_clone_wipe.py (hunk 2 above)
Comment (will-Do) [inline libvirt/tests/src/virsh_cmd/volume/virsh_vol_clone_wipe.py]: As there's no prepare steps above, so it's no need to do cleanup

Comment (kylazhang) [inline libvirt/tests/src/virsh_cmd/volume/virsh_vol_clone_wipe.py]: If pool exists, it is not need to destroy/undefine it?

Comment (will-Do) [inline libvirt/tests/src/virsh_cmd/volume/virsh_vol_clone_wipe.py]: Yes, that should be done by the previous case, we suppose the testing env is clean as we can't do every cleanup steps before running the case.

Comment (kylazhang) [inline libvirt/tests/src/virsh_cmd/volume/virsh_vol_clone_wipe.py]: ok, then no problem

**Comment by Hao-Liu** [comment]
ACK and merge this, Thanks @will-Do @kylazhang

**Pull request merged**